### PR TITLE
javacript1/week3 homework

### PR DESCRIPTION
**Pull request by garyaldahoul**

### CODE REVIEW

File: javascript/javascript1/week3/index.js [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+// remvoe name from array and display the new list of names
+//names.splice(0, 1);
+function createNewArray() {
+  showNewNamesArray.innerHTML = "";
+  for (indexNumber in names) {
+    if (
+      names[indexNumber].toLocaleLowerCase() ===
+      nameToRemove.value.toLocaleLowerCase()
+    ) {
+      names.splice(indexNumber, 1);
```
Comment (sowmya1408) [inline javascript/javascript1/week3/index.js:74]: perfect 💯

File: javascript/javascript1/week3/index.js [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+
+function calculateTime(travelInformation) {
+  travelInformation.speed = speed.value;
+  travelInformation.destinationDistance = destinationDistance.value;
+  const time = travelInformation.destinationDistance / travelInformation.speed;
+  const hour = Math.floor(time);
+  const minutes = Math.floor((time - hour) * 60);
+  showTime.innerHTML = `${hour} Hour and ${minutes} Minutes`;
+  showTime.classList.add("show");
+  return showTime;
```
Comment (sowmya1408) [inline javascript/javascript1/week3/index.js]: Variable naming conventions , hour/minute - is a general naming, be very specific.
ex: travelInHour etc...

I think the function returns the html node, instead it should return  `${hour} Hour and ${minutes} Minutes´, so its a good idea to store it in a variable and use it in the places you would like to :)

File: javascript/javascript1/week3/index.js [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+      Math.floor(seriesDuration[i].hours * 60) +
+      Math.floor(seriesDuration[i].minutes);
+
+    const yourLifeInminutes = yearsToLife.value * 525600;
+    const avrageForEverySeries = (
+      (yourSpendTime * 100) /
+      yourLifeInminutes
+    ).toFixed(3);
+    minutesSeriesTimeArray.push(yourSpendTime);
+    let totalMinutes = minutesSeriesTimeArray.reduce((a, b) => a + b, 0);
```
Comment (sowmya1408) [inline javascript/javascript1/week3/index.js]: Its good to see that you already use Array function like reduce, but
parameter names should be given a good naming convention, should not be used as a, b.

File: javascript/javascript1/week3/index.js [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+  for (let i = 0; i < notes.length; i++) {
+    if (notes[i].id === id) {
+      specificNote = notes[i].content;
+    }
+  }
+
+  showSpecificNote.innerHTML = `The specifice note you want is : ${specificNote}`;
+  showSpecificNote.style.color = "green";
+
+  return specificNote;
```
Comment (sowmya1408) [inline javascript/javascript1/week3/index.js:233]: 👍

**Review by sowmya1408 — APPROVED** [review]
Great job !!! 👍 
Nice to see that you already use array function.
Just follow variable naming convention which should be specific.

File: javascript/javascript1/week3/index.js (hunk 3 above)
Comment (garyaldahoul) [reply javascript/javascript1/week3/index.js]: Hi sowmya,
Thank you so much, I will do change for me code but for reduce((a,b)=>a+b,0), it likes build in function, whatever we put nothing will make it sense. EX:(firstNumber,secondNumber)

Comment (sowmya1408) [reply javascript/javascript1/week3/index.js]: Yes, reduce is a build in function, but the parameter will make sense if write it in this case as 
Ex: (seriesMinutesAcc, seriesMinutesValue)
Acc* - accumulator

File: javascript/javascript1/week3/index.js (hunk 2 above)
Comment (garyaldahoul) [reply javascript/javascript1/week3/index.js]: Yes, I know, the point here, we'lll not use it again, we can return time. And even I can do it without return.

File: javascript/javascript1/week3/index.js (hunk 3 above)
Comment (garyaldahoul) [reply javascript/javascript1/week3/index.js]: I did it like this: reduce((accumulator,accumulator)=>accumulator+accumulator,0)

Comment (sowmya1408) [reply javascript/javascript1/week3/index.js]: I think you still did not get my point, to have a describing parameter names in a function
Refer to this document to get better understanding:
https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md

**Pull request merged**